bin/lint

```diff
@@ -1,6 +1,5 @@
 #!/bin/sh
 set -exu
-LABEL='hmt'
 
-docker build -t $LABEL .
-docker run --net=host -it $LABEL sh -c 'npm run lint'
+CONTAINERS=${1-"hmt"}
```

Comment:
You probably dont need this param, doesnt really do much (ie youd never override the container that lint runs in right?)